app/assets/stylesheets/homepage.css.scss

```diff
@@ -71,14 +76,19 @@
     height: 550px;
     background: transparent image-url("homepage.png") no-repeat center -50px;
     background-size: cover;
-  box-shadow: 0px 30px 40px rgba(0,0,0,.5);
-
+  box-shadow: 0px 30px 40px ;
```

Comment:
Declaration should not have a space before the terminating semicolon<br>`0px` should be written without units as `0`